blazor/combobox/how-to/customization.md

```diff
@@ -1,15 +1,16 @@
 ---
 layout: post
-title: Automatically open dropdown in Blazor ComboBox | Syncfusion
-description: Learn here all about Automatically open the dropdown in Syncfusion Blazor ComboBox component and more.
+title: Customization in Blazor ComboBox | Syncfusion
```

Comment:
move this md file inside blazor/combobox folder instead of blazor/combobox/how-to/